server/sql/analysis/clck_view.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- View for Intel(R) Cluster Checker
+--
+DROP VIEW IF EXISTS clck_1;
+CREATE OR REPLACE VIEW clck_1 AS
+    SELECT test_run.duration,
+        0 AS encoding,
+        test_run.exit_value AS exit_status,
+        submit.hostname,
+        ''::text AS node_names,
+        0 AS num_nodes,
+        test_run.test_run_command_id::character varying AS optionid,
+        test_run.ctid AS rowid,
```

Comment:
Yeah, since we are being pushed to move away from posgresql, I would prefer to not use a system constant here. `test_run_id` should be sufficient.